docker/opa-policies/terraform.rego

```diff
@@ -57,7 +61,15 @@ score = s {
             mod := crud["modify"] * num_modifies[resource_type];
             x := del + new + mod
     ]
-    s := sum(all)
+    others := [ x |
+            some resource_type
+            crud := {"delete": 100, "create": 1, "modify": 1};
```

Comment:
Should we have 100 for undefined objects?
If so is it worth removing all the defined objects that doesn't have other values then default?

Reply:
Maybe for the future